devicon.json

```diff
@@ -8438,15 +8438,19 @@
             "svg": [
                 "original",
                 "original-wordmark",
+                "plain",
+                "plain-wordmark",
                 "line",
                 "line-wordmark"
             ],
             "font": [
+                "plain",
+                "plain-wordmark",
                 "line",
                 "line-wordmark"
             ]
         },
-        "color": "#c51850",
+        "color": "#231f20",
```

Comment:
Why was the color changed to gray here? Seems like `#c51850` should be used, since it's the most dominant color in the original icon. Alternatively the gray (`#050606`) or green (`#63c54d`) color from the icon